src/main/java/uk/ac/ebi/tsc/portal/api/deployment/service/DeploymentService.java

```diff
@@ -64,6 +66,8 @@ public DeploymentStatus findStatusByDeploymentId(Long deploymentId) {
                 () -> new DeploymentStatusNotFoundException(deploymentId));
     }
 
+    @PostAuthorize(value = "hasAuthority(@webConfiguration.adminAuthority())" +
+            " or returnObject.getAccount().getUsername() == authentication.name")
```

Comment:
Is `@adminAuthority` not enough, if it as a bean? Also you can use property notation for `returnObject.account.username`.

Reply:
Now I have changed it to a bean and used it directly.